tests/core/daemon/test_daemon.py

```diff
@@ -777,3 +777,49 @@ async def test_key_label_methods(
     keychain.add_private_key(test_key_data.mnemonic_str(), "key_0")
     await ws.send_str(create_payload(method, parameter, "test", "daemon"))
     assert_response(await ws.receive(), response_data_dict)
+
+
+@pytest.mark.asyncio
+async def test_bad_json(daemon_connection_and_temp_keychain: Tuple[aiohttp.ClientWebSocketResponse, Keychain]) -> None:
```

Comment:
Do we need the temp keychain for these tests?

Reply:
Mm, I was thinking this triggered extra setup work but it seems maybe it doesn't.

https://github.com/Chia-Network/chia-blockchain/blob/69a91d24ef1ef60e1e88de353857c8f0d641b02c/tests/conftest.py#L599-L602